package.json

```diff
@@ -64,6 +64,7 @@
     "i18n-abide": "0.0.23",
     "jsxgettext-recursive": "0.0.5",
     "load-grunt-tasks": "3.2.0",
+    "lodash": "4.6.1",
```

Comment:
You need to re-run shrinkwrap.

Steps to do so:

1. rm -rf node_modules
1. rm npm-shrinkwrap.json
1. npm install --production
1. npm shrinkwrap
1. commit.
1. npm install (to get the dev deps again).